requirements/local.txt

```diff
@@ -1,4 +1,4 @@
 -r base.txt
-ipython
+ipython==1.0.0
```

Comment:
To work with Python 3.2, which is what Ubuntu 12.04 ships for python 3.  "IPython supports Python 2.7 and 3.3 or newer. Our older 1.x series supports Python 2.6 and 3.2." :(